app/components/OceanInterface/OceanInterface.tsx

```diff
@@ -130,6 +132,12 @@ export function OceanInterface (): JSX.Element | null {
             Logging.error(e)
             title = 'Sent but not confirmed'
           }
+
+          notificationObj = {
+            title,
+            body: `Txid: ${transaction.tx.txId}`
```

Comment:
```suggestion
            body: `Transaction ID: ${transaction.tx.txId}`
```

Need to translate `Transaction ID` also